py-bin/tlbin/contracts.json

```diff
@@ -0,0 +1 @@
+../contracts.json
```

Comment:
This is a simlink right? did you try if this correctly works with packing into a dist?

Reply:
Yes, it does work.